src/tm2/tm-device.cpp

```diff
@@ -1439,6 +1466,9 @@ namespace librealsense
         _sensor->register_option(rs2_option::RS2_OPTION_GAIN, std::make_shared<gain_option>(*_sensor));
         _sensor->register_option(rs2_option::RS2_OPTION_ENABLE_AUTO_EXPOSURE, std::make_shared<exposure_mode_option>(*_sensor));
 
+        _sensor->register_option(rs2_option::RS2_OPTION_ENABLE_MAPPING,        std::make_shared<tracking_mode_option<perc::SIXDOF_MODE_ENABLE_MAPPING,        perc::SIXDOF_MODE_NORMAL        >>(*_sensor, "Use an on device map"));
+        _sensor->register_option(rs2_option::RS2_OPTION_ENABLE_RELOCALIZATION, std::make_shared<tracking_mode_option<perc::SIXDOF_MODE_ENABLE_RELOCALIZATION, perc::SIXDOF_MODE_ENABLE_MAPPING>>(*_sensor, "Use appearance based relocalization (depends on mapping)"));
+        _sensor->register_option(rs2_option::RS2_OPTION_ENABLE_POSE_JUMPING,   std::make_shared<tracking_mode_option<perc::SIXDOF_MODE_ENABLE_JUMPING,        perc::SIXDOF_MODE_ENABLE_MAPPING>>(*_sensor, "Allow pose jumping (depends on mapping)"));
```

Comment:
Is jumping reliant on mapping only or also on relocalization (tracking) ?

Reply:
Jumping and relocalization are both separately reliant on mapping and not on each other.